cmd/node/node.go

```diff
@@ -294,16 +295,18 @@ func (app *App) Initialize() (err error) {
 			log.Int("hare_wakeup_delta", app.Config.HARE.WakeupDelta),
 			log.Int("hare_limit_iterations", app.Config.HARE.LimitIterations),
 			log.Int("hare_round_duration", app.Config.HARE.RoundDuration))
-		return errors.New("incompatible tortoise hare params")
+
+		desc := "layer-duration-sec * tortoise-zdist <= hare-wakeup-delta + (1 + hare-limit-iterations * 4) * hare-round-duration-sec"
+		return log.ErrTortoiseHareParams(desc)
```

Comment:
please no need to invent something weird with these wrappers. 

there are errors that have custom handling in smapp and they should have code, for every other error code is not needed and smapp should provide only information available in the error. we definitely won't be adding code to every possible error that is returned across the stack.

another thing to consider that some errors will want to log their details in a structured way instead of flat string. those errors should implement interface the way it was described above.